files/galaxy/dynamic_job_rules/production/total_perspective_vortex/tools.yml

```diff
@@ -3696,7 +3697,28 @@ tools:
       - interactive_pulsar
       accept:
       - training-exempt
+  restricted_interactive_tool:
+  inherits: interactive_tool_.*
+  rules:
+    - id: restricted_interactive_tool_restriction_rule
+      if: |
+        from datetime import datetime, timedelta
+        interactive_tool_embargo_days = 28
+        may_use_ITs = False
+        if 'Trusted_for_ITs' in [r.name for r in user.all_roles() if not r.deleted]:
+          may_use_ITs = True
+        if any([r.name.startswith('training') for r in user.all_roles()]):
+          may_use_ITs = True
+        threshold = datetime.now() - timedelta(days=interactive_tool_embargo_days)
+        if user.create_time < threshold:
+          may_use_ITs = True
+        not may_use_ITs
+      fail:
+        This tool is currently not available for your user profile. Please email help@genome.edu.au if you need access or have any questions.
```

Comment:
The error doesn't indicate that the user will actually be able to run the tool automatically in future, or when. Should we say something like:
You must wait {X} days before being able to use this tool. Please email help@genome.edu.au if you need access sooner or have any questions.